### ISSUE TITLE
rqs.Batch typescript response seems wrong

### ISSUE THREAD

**Issue by spirift** [recombee/node-api-client#22]
Hi Recombee team. Thanks so much for adding all the types in the latest release. 

I think I found an issue with the response type for the Batch request. I'm performing a batch of SetItemValues and can see a response of 
```typescript
{ code: number, json: string }[]
```

but this does not match with the BatchResponse type

```typescript
type BatchResponse = {
    statusCode: number;
    response: Response[];
  }
```

It's easy enough to override this but I thought you would want to know :)

**Comment by vallpaper** [comment]
Thank you for the report 👍 we will try to release the fix soon-ish :)

**Comment by DanielBreiner** [comment]
Fixed in [v4.1.4](https://github.com/recombee/node-api-client/releases/tag/v4.1.4), thanks for the report.
